### DISCUSSION TITLE
Help with new Channels feature, please...

### DISCUSSION THREAD

**Discussion by aleixdorca** [open-webui/open-webui#8164] [Q&A] [answered]
Hi,

I've been trying the new Channels feature. I can create Channels, assign groups and use it like a chat between group members.

For the life of me, I have not found how to send a message to the AI model.

Help, please?

Cheers,

Aleix.

**Comment by tjbck** [comment, marked as answer]
Has to be externally hosted with https://github.com/open-webui/bot for now, but built-in integration will be added soon.

**Reply by aleixdorca** [reply]
Thanks for that, Timothy.

**Reply by razzium** [reply]
@aleixdorca where do you find UI part to create Channels, assign groups and use it like a chat between group members ?
I mean, I activated channels in Settings but I see nothing related to channels in the UI after the change.

**Reply by aleixdorca** [reply]
@razzium Once you activate the Channels (Beta) setting (and maybe refresh the UI) you will see a new addition to the left bar called "Channels".

Here you can create Channels if you are an admin user. Plain users can only see Channels they are part of.

Groups have to be added, as before, in the Users' Settings menu. These groups can be, then, added to a Channel.

Hope this helps.

Aleix.

**Reply by razzium** [reply]
Thanks for your answer @aleixdorca.
So visibly I have an issue, I activated it in Admin but not able to see a new addition to the left bar called "Channels".

**Reply by aleixdorca** [reply]
Have you tried doing a "Force Reload" in your browser. In Mac's Chrome is Command+Shift+R, for example. That usually does the trick for me.

**Reply by razzium** [reply]
Yes I tried but no success.
Issue come from the line 657 of "Sidebar.svelte" file:
`{#if $config?.features?.enable_channels && ($user.role === 'admin' || $channels.length > 0) && !search}`

If I print "$config.features" there is no "enable_channels":
{
    "auth": true,
    "auth_trusted_header": false,
    "enable_ldap": false,
    "enable_api_key": true,
    "enable_signup": true,
    "enable_login_form": true,
    "enable_websocket": true
}

**Comment by vivekpd15** [comment]
Love this feature. Any update on when this would become integrated? @tjbck

**Reply by bess-trailer** [reply]
Probably the biggest step forward for collaboration. 
I haven't seen it available anywhere.  Highly anticipated!!!

**Reply by flefevre** [reply]
yes waiting for this feature making the bridge between human and bots !

**Comment by Taikono-Himazin** [comment]
Are there any updates on this feature?
I'm really looking forward to it.

Can I think of the channel feature as a way to talk in a group with a bot embedded in it?

**Comment by okaeiz** [comment]
Can we mention the models with `@`? For instance, we may type `@gemini-2.5-pro` and then start talking to the model.

**Reply by flefevre** [reply]
I have tried to describe deeper what could be the channel feature with agents here

https://github.com/open-webui/open-webui/discussions/14588

Hope it could help in the design of the future roadmap
